app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/DataCreationActivity.kt

```diff
@@ -9,9 +9,9 @@ import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
```

Comment:
I think you could address the warning in this file using https://stackoverflow.com/questions/24832497/avoid-passing-null-as-the-view-root-need-to-resolve-layout-parameters-on-the-in